build.sbt

```diff
@@ -48,6 +48,13 @@ lazy val hbase = project
     fork in Test := true
   )
 
+lazy val ironmq = project
+  .enablePlugins(AutomateHeaderPlugin)
+  .settings(
+    name := "akka-stream-alpakka-ironmq",
+    Dependencies.IronMq
+  )
+
```

Comment:
Also add `ironmq` to the root `alpakka` project at the top of this file. Otherwise no ironmq connector tests are ran by Travis.